frontend/src/views/Annotate.vue

```diff
@@ -188,13 +189,20 @@ export default {
       showLeaveProjectModal: false,
       showStageIntroCard: false,
       showThankyouCard: false,
+      annotationRendererTransitionEnabled: false,
     }
   },
 
   methods: {
     ...mapActions(["getUserAnnotationTask", "getUserAnnotationTaskWithID",
       "completeUserAnnotationTask", "rejectUserAnnotationTask", "annotatorLeaveProject",
       "changeAnnotation"]),
+    triggerAnnotationRendererTransition(){
+      this.annotationRendererTransitionEnabled = true
+      setTimeout(()=>{
+        this.annotationRendererTransitionEnabled = false
+      }, 500)
```

Comment:
Is it possible to just declare an `animationend`/`animationcancel` event handler that sets `this.annotationRendererTransitionEnabled = false`, then we don't need to worry about the `setTimeout` every time?  But if not it's fine as it is.